canonical/lxd-ui
# Add airgap support

## Done

- embed font, so in airgapped environments we don't try to load from assets.ubuntu.com
- avoid refetch of image index, so the image list appears immediately if the indexes can't be fetched
- remove unused icon

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check the ui for any third party requests. The font should be loaded locally. When blocking other domains, the image selector should load immediately
